src/modules/logger/logged_topics.cpp

```diff
@@ -123,6 +122,7 @@ void LoggedTopics::add_default_topics()
 	add_topic_multi("control_allocator_status", 200, 2);
 	add_optional_topic_multi("rate_ctrl_status", 200, 2);
 	add_topic_multi("sensor_hygrometer", 500, 4);
+	add_topic_multi("rpm", 200, 8);
```

Comment:
```suggestion
	add_topic_multi("rpm", 200);
```

This will debug to ORB_MULTI_MAX_INSTANCES (usually 10).

Reply:
I would also change `add_topic_mult` -> `add_optional_topic_multi` (small logger memory optimization).

Reply:
Changed to 'Optional multi topic logger'.